src/shared/sam/localLambdaRunner.ts

```diff
@@ -320,6 +321,7 @@ export async function invokeLambdaFunction(
 
         // sam local invoke ...
         const command = new SamCliLocalInvokeInvocation(localInvokeArgs)
+        const samVersion: string = await getSamCliVersion(getSamCliContext()).then(v => v).catch()
```

Comment:
We almost always want to avoid silent failure, it can be painful. Suggest logging the exception in catch()